packages/api/src/external/commonwell/organization.ts

```diff
@@ -76,22 +90,28 @@ export async function organizationToCommonwell(
   return cwOrg;
 }
 
-export async function get(orgOid: string): Promise<CWOrganization | undefined> {
-  const { log, debug } = out(`CW get org oid ${orgOid}`);
-  const commonWell = makeCommonWellAPI(Config.getCWMemberOrgName(), Config.getCWMemberOID());
+export async function get(cxId: string, orgOid: string): Promise<CWSdkOrganization | undefined> {
+  const { log, debug } = out(`CW get (Organization) - CW Org OID ${orgOid}`);
+  if (!(await isCWEnabledForCx(cxId))) {
+    log(`CW disabled for cx ${cxId}, skipping...`);
+    return undefined;
```

Comment:
If we return undefined here then in the  `"/organization/:oid",` we will throw the error `Organization not found`. Wondering if we could handle this a bit different as i assume this is for ops dash so we can be more descriptive that cw is disabled.

Reply:
No longer relevant. FF check is removed.